.github/workflows/autobuild.yaml

```diff
@@ -152,6 +152,8 @@ jobs:
             "-DCMAKE_CXX_COMPILER=cl"
             "-DLLVM_ENABLE_ZLIB=OFF"
             "-DLLVM_USE_CRT_RELEASE=MT"
+            "-DLLVM_ENABLE_PDB=ON"
+            "-DLLVM_PARALLEL_LINK_JOBS=1"
```

Comment:
Note I'm turning the parallel link jobs for Windows builder to 1 since there's only 7 GiB available for [Github's free action plan](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources). Running these jobs together would somehow hit this bottleneck and results in [OOM](https://github.com/zyn0217/clangd/actions/runs/6439605610/job/17487472454).

And here's the [action link](https://github.com/zyn0217/clangd/actions/runs/6440816610) to my successfully built [artifacts](https://github.com/zyn0217/clangd/releases/tag/17.0.2). (Windows only)